make.xsh

```diff
@@ -492,6 +517,31 @@ def export_rootski_profile_aws_creds():
         )
 
 
+def export_rootski_profile_aws_creds_db_backup():
+    aws_creds_rel_path = ".aws/credentials"
+    aws_credentials_file = Path.home() / aws_creds_rel_path
+    if not aws_credentials_file.exists():
+        log(
+            f"No credentials file found at \"{aws_creds_rel_path}\". Credentials not exported."
+            + " This is okay if you don't need valid HTTPS certs for Traefik or if you are using IAM"
+            + "Role-based Authentication.",
+            mode="error"
+        )
+
+    try:
+        config = configparser.RawConfigParser()
```

Comment:
Fancy! Did you set up this `configparser` logic? Or did I already have it somewhere else and you moved it?

Reply:
more copy paste but maybe since I had to find it, understand what it did, and then copy and paste it, I might as well have set it up myself right? jk. definitely found and copied